src/plone/restapi/types/adapters.py

```diff
@@ -32,6 +31,15 @@
 from plone.restapi.types.utils import get_fieldsets
 from plone.restapi.types.utils import get_jsonschema_properties
 
+import pkg_resources
```

Comment:
You could move the `ChoiceslessRelationListSchemaProvider` into e separate module. Then you could avoid the conditional import stuff by just using a ZCML condition.